presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoDriver.java

```diff
@@ -68,7 +73,7 @@
 
     private final JettyIoPool jettyIoPool;
 
-    private final ReferenceCountingLoadingCache<Map<Object, Object>, QueryExecutor> queryExecutorCache;
+    private final ReferenceCountingLoadingCache<HttpClientConfig, QueryExecutor> queryExecutorCache;
```

Comment:
As a temporary workaround you can extend the `HttpClientConfig` inline and implement those method  there.